client/components/organization-wrapper/organization-wrapper.js

```diff
@@ -349,7 +359,7 @@ OrganizationWrapper.propTypes = {
     configuration: PropTypes.shape({
       title: PropTypes.string,
       pageTitle: PropTypes.string,
-      css_path: PropTypes.string,
+      css_path: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
```

Comment:
we can leave css_path as a string for backward compatibility and then add `css` to be only an array.